app/src/main/java/org/amfoss/templeapp/activities/ReadSingleData.java

```diff
@@ -50,14 +50,14 @@ public void radioGroup(RadioGroup group, int checkedId) {
                 case R.id.radio_donate:
                     binding.totalView.setVisibility(View.VISIBLE);
                     flag = 0;
-                    Toast.makeText(getBaseContext(), getString(R.string.selected_donate), Toast.LENGTH_SHORT)
+                    Toast.makeText(getBaseContext(), getResources().getString(R.string.search_donations), Toast.LENGTH_SHORT)
```

Comment:
Is `getResources() ` really needed? Will it not work without `getResources()`

Reply:
Will remove it